iss-55 topページの作成

koala777 opened the pull request:
未ログイン時のトップページ(/) にアクセスしたときのページを作成する

koala777 committed:
iss-55 topページの作成

未ログイン時のトップページ(/) にアクセスしたときのページを作成する

causeway commented:
2/2 日に　Rails 側が変更になりました。
TimeCard のリンクは、 Root 
Project は未ログイン状態でも表示するようになっています。
お手数ですが、対応お願い致します。

koala777 committed:
iss-55 topページの作成

PR指摘反映
・"TimeCard"のリンクをTopに変更
・"Projects"リンクを設定
・Projects/indexはAuth対象外とする設定を追加

koala777 commented:
Projectは未ログインでも表示するように変更しました。
ご確認をよろしくお願いいたします。
